common/src/main/java/net/dodogang/plume/block/PlumeCraftingTableBlock.java

```diff
@@ -21,11 +21,9 @@ public PlumeCraftingTableBlock(Settings settings) {
     @Override
     public NamedScreenHandlerFactory createScreenHandlerFactory(BlockState state, World world, BlockPos pos) {
         return new SimpleNamedScreenHandlerFactory(
-                (syncId, inventory, player) -> new PlumeCraftingScreenHandler(
-                    syncId,
-                    inventory,
-                    ScreenHandlerContext.create(world, pos),
-                    state.getBlock()
-                ), TITLE);
+            (syncId, inventory, player)
+                -> new PlumeCraftingScreenHandler(
+                    syncId, inventory, ScreenHandlerContext.create(world, pos), state.getBlock()),
+            TITLE);
```

Comment:
I don't like this.

Reply:
ew that ruins every principle of lambda styling